src/Mailjet/Response.php

```diff
@@ -150,7 +150,7 @@ protected function decodeBody($body)
              */
             $maxIntLength = strlen((string) PHP_INT_MAX) - 1;
             $jsonWithoutBigIntegers = preg_replace('/:\s*(-?\d{'.$maxIntLength.',})/', ': "$1"', $body);
-            $object = json_decode($jsonWithoutBigIntegers);
+            $object = json_decode($jsonWithoutBigIntegers, true);
```

Comment:
I think it should be : 
`$object = json_decode($jsonWithoutBigIntegers, true, 512);`